tensorflow/python/framework/tensor_shape_div_test.py

```diff
@@ -35,6 +35,15 @@ def testDivSucceeds(self):
         for y in values:
           self.assertEqual((x / y).value, (x // y).value)
 
+  def testRDivFail(self):
+    # Note: This test is related to GitHub issue 25790.
+    """Without from __future__ import division, __rdiv__ is used."""
+    if six.PY2:  # Old division exists only in Python 2
+      two = tensor_shape.Dimension(2)
+      message = (r"unsupported operand type\(s\) for \/: "
+                 r"'int' and 'Dimension', please use \/\/ instead")
```

Comment:
are you sure you need to quote the forward slashes?

Reply:
Thanks @drpngx. I tried locally indeed they are not needed. The PR has been updated.